lib/cia/auditable.rb

```diff
@@ -6,7 +6,15 @@ def self.included(base)
     end
 
     def cia_changes
-      changes
+      if respond_to?(:has_changes_to_save?)
+        if has_changes_to_save?
+          changes_to_save
+        else
+          saved_changes
+        end
+      else
+        changes
+      end
```

Comment:
http://ashleyangell.com/2019/05/changes-to-activerecorddirty-in-rails-5-2/

for future reference.